sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -670,6 +681,41 @@ export class KeyClient {
     });
   }
 
+  /**
+   * Releases a key from a managed HSM.
+   *
+   * The release key operation is applicable to all key types. The operation requires the key to be marked exportable and the keys/release permission.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(vaultUrl, credentials);
+   * let result = await client.releaseKey("myKey", "keyVersion", target)
+   * ```
+   *
+   * @param name - The name of the key.
+   * @param version - The version of the key.
+   * @param options - The optional parameters.
+   */
+  public releaseKey(
+    name: string,
+    version: string,
```

Comment:
`version` is functionally optional like with other endpoints. Confirmed with Hervey.

Reply:
Moved to the options bag and made optional. Thanks for confirming!